membership.go

```diff
@@ -0,0 +1,41 @@
+package civogo
+
+import (
+	"bytes"
+	"encoding/json"
+)
+
+// MembershipResponse is the response for the memberships of a user
+type MembershipResponse struct {
```

Comment:
`Account` and `Organisation` DTOs used here contain a lot more fields than the ones provided by `v2/memberships`. Shouldn't you use `MembershipAccount` and `MembershipOrganisation` defined below?